Makefile

```diff
@@ -116,6 +116,7 @@ GINKGO_FOCUS ?= Workload cluster creation
 GINKGO_SKIP ?= Creating a GPU-enabled cluster
 GINKGO_NODES ?= 3
 GINKGO_NOCOLOR ?= false
+GINKGO_ARGS ?= -stream --progress
```

Comment:
Are we adding these Ginkgo flags for all the `test-e2e-*` targets? I like them, but I thought we didn't want to stream in the real CI context, so the test output isn't interleaved.

Reply:
oh I see what you're saying #1020 only added it to local e2e... will do the same